### DISCUSSION TITLE
how to use dispatch in volt component based class?

### DISCUSSION THREAD

**Discussion by yacobii** [livewire/livewire#6423] [1. Help]
In Volt how I can get dispatched from another Volt component? where I must use #[On('refresh')]?

<?php
 
use Livewire\Volt\Component;
 
new class extends Component {

} ?>
 
<div>
</div>

**Comment by joshhanley** [comment]
@mehdiyaq Please ask on the volt repo 🙂

**Reply by yacobii** [reply]
volt repo doesn't have discussions!

**Reply by joshhanley** [reply]
@mehdiyaq Yeah but volt isn't managed by us, but is managed by the Laravel team
